### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi vshn team

I installed k8up and had some difficulties so I improved the docs a bit.

Cheers

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -22,7 +22,8 @@ Then install K8up itself:
 
 [source,bash]
 ----
-helm install appuio/k8up
+kubectl create namespace k8up-operator
+helm install appuio/k8up --namespace k8up-operator --generate-name
```
Comment (srueg) [inline docs/modules/ROOT/pages/getting-started.adoc]: I'd suggest to rather specify a name than generating one
```suggestion
helm install k8up appuio/k8up --namespace k8up-operator
```

**Comment by akosma** [comment]
LGTM!

File: docs/modules/ROOT/pages/getting-started.adoc (hunk 1 above)
Comment (dns2utf8) [reply docs/modules/ROOT/pages/getting-started.adoc]: Sounds good to me

**Comment by dns2utf8** [comment]
🎉 Looking forward to see everything merged

**Pull request merged**